www/js/toolbar.js

```diff
@@ -327,6 +348,11 @@ const syncContent = h('div.sync-content', [
       false,
     ),
   ]),
+  h('div.qr-container', [
```

Comment:
maybe we can get a head start in to componentization and have this in its own file and import it?

Reply:
Umm, yes, that's part of the toolbar refactor, all this is going to separate files anyway.